web/src/components/timeControls.js

```diff
@@ -63,15 +64,36 @@ const DateOptionWrapper = styled.div`
   padding-top: 8px;
 `;
 
-const options = [
-  { key: 'day', label: 'time-controller.day' },
-  { key: 'month', label: 'time-controller.month' },
-  { key: 'year', label: 'time-controller.year' },
-  { key: '5year', label: 'time-controller.5year' },
-];
+const getOptions = (language) =>
+  useMemo(
```

Comment:
I don't think it makes sense to useMemo here - in general it should only be used for heavy calculations, as the performance overhead of using the functionality is otherwise higher than not using it :)

Reply:
Nevermind, you already fixed this!